src/components/canvas/text-sidebar/index.tsx

```
@@ -0,0 +1,234 @@
+    [updateTextProperty],
+  );
+
+  if (!isOpen || !selectedTextShape) {
+    return null;
+  }
+
+  return (
+    <div
+      className={cn(
+        'fixed right-5 top-1/2 transform -translate-y-1/2 w-80 backdrop-blur-xl bg-white/[0.08] border-white/[0.12] gap-2 p-3 saturate-150 border rounded-lg z-50 transition-transform duration-200',
+        true ? 'translate-x-0' : 'translate-x-full',
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)